src/nedap/utils/test/impl.cljc

```diff
@@ -40,3 +41,22 @@
        (if (cljs.test/successful? summary)
          (set! (.-exitCode js/process) 0)
          (set! (.-exitCode js/process) 1)))))
+
+(spec/def ::to   some?)
+(spec/def ::from some?)
+(spec/def ::to-change some?)
+(spec/def ::expect-options
```

Comment:
Probably better to discard `::expect-options` in favor of individually validating destructured args.

That way, we ensure destructuring was correct